web/src/engine/websites/iqiyi.ts

```diff
@@ -0,0 +1,46 @@
+import { Tags } from '../Tags';
+import icon from './iqiyi.webp';
+import { Chapter, DecoratableMangaScraper, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as MH from './decorators/MH';
+import { FetchJSON, FetchRequest } from '../FetchProvider';
+
+type APIChapters = {
+    data: {
+        episodes: {
+            episodeId: number,
+            episodeOrder: number,
+            episodeTitle : string
+        }[]
+    }
+
+}
+function MangaExtractor(anchor: HTMLAnchorElement) {
+    const id = anchor.pathname;
+    const title = anchor.nextElementSibling.textContent.trim();
+    return { id, title };
+}
+
+@Common.MangaCSS(/^{origin}\/manhua\/detail_\S+\.html$/, '.detail-tit h1')
+@Common.MangasMultiPageCSS('/manhua/category/%E5%85%A8%E9%83%A8_-1_-1_9_{page}', 'ul.cartoon-hot-ul li.cartoon-hot-list a.cartoon-cover', 1, 1, 0, MangaExtractor)
+@Common.PagesSinglePageCSS('ul.main-container li.main-item img', MH.PageLinkExtractor)
```

Comment:
This page looks not like `MH`, does the default `PageLinkExtractor` from `Common` not work?

Reply:
1) iqiyi is extending MH in Hakuneko so it ended up in that PR, i think there is a reason for that.
2) No it wouldnt work

![image](https://github.com/manga-download/haruneko/assets/10671518/df037176-2f14-479b-bed5-634416ad7f30)